pkg/utils/utils.go

```diff
@@ -21,6 +21,15 @@ import (
 	"log"
 	"os"
 	"path"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	getTermios = 0x5401 // syscall.TCGETS
+	setTermios = 0x5402 // syscall.TCSETS
```

Comment:
@iaguis Both are done. Changed the const to `syscall.TCGETS`, and removed `setTermios`.